README.md

```diff
@@ -1 +1,360 @@
+
+### Payment Flow
+
+#### Payment flow using Pocket Vouchers
+
+```
+requests.create > Pocket Voucher Code Entry screen on terminal > User enters Pocket Voucher Code into terminal > requests.pay > terminal Loading >Webhook Notification > terminal display result.
+```
+
+#### Payment flow using QR Code
+
+```
+requests.create > Generate QR Code on terminal > User selects payment type on phone > Phone Scans > requests.pay > Authenticated and Processed via Phone > Webhook Notification > terminal display result.
+You will need to create a request ‘/requests.create’ first and use the requestId for making a payment ‘/requests.pay’.
+```
+
+### Glossary
+
+### API keys and access
+
+To gain access to the payments API, please create an account in our Portal. Once you’ve completed the sign up process and acknowledged our terms, we will provide a merchant_id, secret, and api_key via the Portal.
+
+To generate test credentials, click here.
+
+
+### API protocols
+
+In order to be authorized to use any of the payments API endpoints, you will need to send your api_key in the header of your request.
+
+
+### API hosts
+
+service.centrapay.com/payments/api
+
+### Endpoints
+
+|             |                                                                                                   |
+|-------------|---------------------------------------------------------------------------------------------------|
+| Requests    | /requests.cancel <br> /requests.create <br> /requests.info <br> /requests.pay <br> /requests.void |
+| Service     | /service.info                                                                                     |
+| Transaction | /transactions.processSettlements <br> /transactions.refund                                        |
+
+### QR code payments
+
+#### Merchant Push
+
+At a high level, a merchant makes a **POST** request to `/requests.create` and optionally sets notifyURL to the URL that will receive webhook notifications when the request status is updated. 
+
+A response will be sent to the merchant with a payment request and accompanying QR code (as a base64 encoded string) that encodes the requestId, which can then be presented to consumers as the merchant sees fit.
+
+ A consumer then is then presented with the ID or QR code, which they then scan to initiate payment. If the payment is successful, a notification is sent to the webhook (if provided), confirming with the merchant that the consumer has completed the payment request.
+
+#### Merchant Pull
+
+Where the merchant does not have a screen or payment terminal to display the QR code, this method works well by allowing merchants to simply scan a 2D barcode which is presented on the consumers device to accept payment.
+
+Steps: 
+1. Generate barcode in Sylo 
```

Comment:
Maybe remove the mention of Sylo, we could define what a 'wallet' is somewhere and can mention we are currently partnered with Sylo but throughout the docs we can just use wallet.